bin/set_environment_variables.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+## set the environment variables for CCD_Data_Store and
+## CCD_definition_Store
+
+function set_env_variables() {
+    if [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+        set_environment_variables_unix
+    elif [ "$(expr substr $(uname -s) 1 5)" == "MINGW" ]; then
+        set_environment_variables_windows
+    fi
+}
+
+function set_environment_variables_unix () {
+  echo "Setting environment variables on Unix."
+  set -a
+     source env_data_store.txt
+     source env_definition_store.txt
+  set +a
+}
+
+function set_environment_variables_windows () {
+  echo "Setting environment variables on Windows."
+  parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+  cd "$parent_path"
+  set_env_variables_from_file "../bin/env_data_store.txt"
```

Comment:
this should be ./env_data_store.txt -- ../bin/ is unnecessary.

Reply:
Removed the /bin from file path.